required/latex-lab/latex-lab-block.dtx

```diff
@@ -1529,6 +1570,7 @@
                        { - \int_use:N \l_@@_inner_level_counter_tl }
                   }
                   \l_@@_unused_blockenv_keys_tl
+                  { #2 }
 %    \end{macrocode}
 %    Again the instance may has processed a few keys from the sofar
```

Comment:
```suggestion
%    Again the instance may have processed a few keys from the sofar
```

Reply:
And "so far" is two words, is it not: or is this only for us oldies??

Reply:
Nothing happened it seems! 
No response?

Reply:
Only that David had made some changes/suggestions to that sentence. 
And so: Reasonably real ... maybe.
Nothing to get in such a tizzie about! 
But not worth fussing about it!